ui/app/templates/permissions_modal.html

```diff
@@ -23,7 +23,7 @@ <h3 class="modal-title" ng-show="showRow()"> Username: {{ user.username }}</h3>
         <form role="form" ng-submit="is_edit ? updatePermission(permission) : addPermission()">
           <div class="form-group" ng-class="{'has-error': errors.permission}">
             <label for="id_permission">Permission</label><br>
-              <select name="singleSelect" class="form-control" id="id_permission" ng-model="permission.permission">
+              <select name="singleSelect" class="form-control" id="id_permission" ng-model="permission.permission" ng-change="onPermission()">
```

Comment:
Have you considered using $scope.$watch on permission.permission instead? It would be more consistent with what we do elsewhere.

Reply:
This is actually better and cleaner, thanks